webapp/src/components/backstage/playbook_runs/playbook_run/rhs_status_updates.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+
+import {PlaybookRun, StatusPostComplete} from 'src/types/playbook_run';
+
+import StatusUpdateCard from './update_card';
+
+interface Props {
+    playbookRun: PlaybookRun;
+    statusUpdates: StatusPostComplete[];
+}
+
+const RHSStatusUpdates = ({playbookRun, statusUpdates}: Props) => {
+    if (playbookRun.status_posts.length === 0) {
+        return null;
+    }
+
+    return (<Container>
+        {statusUpdates.map((post) => (
+            <Wrapper key={post.id}>
+                <StatusUpdateCard post={post}/>
+            </Wrapper>
+        ))}
+    </Container>);
```

Comment:
Nit: would it look better if we moved `Container` on the next line:
```suggestion
    return (
      <Container>
          {statusUpdates.map((post) => (
              <Wrapper key={post.id}>
                  <StatusUpdateCard post={post}/>
              </Wrapper>
          ))}
      </Container>
    );
```